simulatorUser/use.rst

```diff
@@ -7,29 +7,30 @@ The main interface is a script to generate a single image.
 There is also a script that will generate a small dataset for testing the pipeline.
 
 The first time the Simulator is run, it will generate a PSF model and cache it;
-this can about an hour.
+this can take about an hour.
 Subsequent executions will be much faster.
 
+Initial setup
+-------------
 
-Single image
-------------
+``makePfsDesign``
+~~~~~~~~~~~~~~~~~
 
-The ``makeSim`` script runs the Simulator to produce a single image
-and the associated ``pfsConfig`` file.
-An example command-line is::
+The simulator requires a ``pfsDesign`` (or ``pfsConfig``) 
```

Comment:
No, it will only accept a `pfsDesign` file.

Reply:
Fixed.